DataFormats/TotemDigi/interface/TotemTriggerCounters.h

```diff
@@ -0,0 +1,24 @@
+/****************************************************************************
+*
+* This is a part of the TOTEM offline software.
+* Authors: 
+*   Jan Kašpar (jan.kaspar@gmail.com) 
+*    
+****************************************************************************/
+
+#ifndef DataFormats_TotemDigi_TotemTriggerCounters
+#define DataFormats_TotemDigi_TotemTriggerCounters
+
+/**
+ * Trigger counters from LoneG.
+**/
+struct TotemTriggerCounters
+{
+  unsigned char type;
+  unsigned int event_num, bunch_num, src_id;
+  unsigned int orbit_num;
+  unsigned char revision_num;
+  unsigned int run_num, trigger_num, inhibited_triggers_num, input_status_bits;
```

Comment:
On 4/13/16 6:19 AM, jan-kaspar wrote:
> In DataFormats/TotemDigi/interface/TotemTriggerCounters.h
> <https://github.com/cms-sw/cmssw/pull/13837#discussion_r59545502>:
> 
>> +*    
>> +****************************************************************************/
>> +
>> +#ifndef DataFormats_TotemDigi_TotemTriggerCounters
>> +#define DataFormats_TotemDigi_TotemTriggerCounters
>> +
>> +/**
>> + * Trigger counters from LoneG.
>> +**/
>> +struct TotemTriggerCounters
>> +{
>> +  unsigned char type;
>> +  unsigned int event_num, bunch_num, src_id;
>> +  unsigned int orbit_num;
>> +  unsigned char revision_num;
>> +  unsigned int run_num, trigger_num, inhibited_triggers_num, input_status_bits;
> 
> To stay on the pragmatic side: will accept the PR if I change this class
> to struct?

At the minimum I have asked only for the initialization to be added.

class&get/set is a coupled request to make this more C++ better

> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/13837/files/7f46a77fbadfb9757b15e4c9c6b194687f264bc3#r59545502>
>